Day-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,52 @@
 /* All styles for your header Components live here */
+
+.top {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between; 
+    width: 100%;
+    }
+    
+    .imgContainer {
+        display:flex;
```

Comment:
The spacing is also off here. Don't forget to use your extensions to clean up afterward.